package.json

```diff
@@ -17,7 +17,7 @@
     "knip-ci": "knip --no-exit-code --reporter json --config .github/knip.ts",
     "prepare": "husky install",
     "test": "jest --setupFiles dotenv/config --coverage",
-    "worker": "wrangler dev --env dev --port 4000"
+    "worker": "wrangler dev --env dev --port 5000"
```

Comment:
Why?

Reply:
I am trying to replicate this, but I am not able to launch wrangler on port 4000.

Platform:
Wrangler@3.62.0
Node 20.11.1
MacOS

Reply:
@gentlementlegen rfc

Reply:
Don't you have anything running already on port 4000? What error do you get?

Reply:
No, I don't have anything running on port 4000. 

Possible bug in wrangler [Wrangler Issue](https://github.com/cloudflare/workers-sdk/issues/4709)